<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>91</width>
      <height>131</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="delayBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>120</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Delay</string>
    </property>
    <widget class="QRadioButton" name="no_delay_dutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>106</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>No delay</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="with_delay_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>106</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>With delay</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="user_instructions_field">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>user_instructions</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>162</width>
      <height>85</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="time_grid">
     <item row="0" column="0">
      <widget class="QLCDNumber" name="minutes_lcd"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="seconds_label">
       <property name="text">
        <string>seconds</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="seconds_lcd"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="minutes_label">
       <property name="text">
        <string>minutes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pause_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="points_grid">
     <item row="0" column="0">
      <widget class="QLCDNumber" name="points_lcd"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="points_label">
       <property name="text">
        <string>points</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
